docs/formats/zng.md

```diff
@@ -627,19 +592,19 @@ where `<nelem>` and each symbol name is encoded as in an enum typedef.
 A map type value has the form:
 ```
 ----------------------------
-|0x23|<key-type>|<val-type>|
+|0x24|<key-type>|<val-type>|
 ----------------------------
 ```
 where `<key-type>` and `<val-type>` are recursive encodings of type values.
 
-#### 4.7 Alias Type Value
+#### 4.7 Type Name Type Value
 
-An alias type value may appear either as a definition or a reference.
-When an alias appears in reference form, it must have been previously
+A type name type value may appear either as a definition or a reference.
+When a type name appears in reference form, it must have been previously
 defined in the type value in accordance with a left-to-right depth-first-search (DFS)
 traversal of the type.
```

Comment:
Also wondering if this goes in zed-data-model since this is a characteristic of named types that applies to all implementations.

Reply:
The data model just says types can be named and bindings are local in scope.  It's up to the serialization formats to decide how to implement this.  e.g., there is nothing about a DFS order for type bindings that belongs in the data model.